src/ui/MLFacet/MLFacetOptions.ts

```diff
@@ -44,13 +47,34 @@ export const MLFacetOptions = {
    * Specifying a value for this option is required for the `MLFacet` component to work.
    */
   field: ComponentOptions.buildFieldOption({ required: true, section: 'CommonOptions' }),
+  /**
+   * Specifies the criteria to use to sort the facet values.
+   *
+   * See {@link FacetSortCriteria} for the list and description of possible values.
+   *
+   * Default value is the `undefined`
+   */
+  sortCriteria: ComponentOptions.buildStringOption({
+    postProcessing: value => (isFacetSortCriteria(value) ? value : undefined),
```

Comment:
What do you think of converting `isFacetSortCriteria` to `sanitizeFacetSortCriteria` and rather than returning `undefined`, returning a good default value, e.g. `occurences`?

Reply:
by default the API is going to get undefined, and follow through with the UX recommendation

Reply:
Then I guess we should be explaining in the documentation what happens if you don't specify one.